medusa-example.ini

```diff
@@ -60,6 +60,14 @@ storage_provider = <Storage system used for backups>
 ; storage_provider should be either of "local", "google_storage", "azure_blobs" or the s3_* values from
 ; https://github.com/apache/libcloud/blob/trunk/libcloud/storage/types.py
 
+storage_class = <Storage Class Name used to store backups>
+; AWS S3 Storage classes: STANDARD | REDUCED_REDUNDANCY | STANDARD_IA | ONEZONE_IA | INTELLIGENT_TIERING | GLACIER | DEEP_ARCHIVE | GLACIER_IR
```

Comment:
Starting from `GLACIER`, things start to break.
The issue is that we can upload stuff with this storage class, but the we try to read stuff back, and we cannot . At least not in the current implementation.
I think the easy way out is to simply not support the 3 lowest tiers. We 'd need to document this somewhere, and then add checks in the config.py to not allow specifying these.